templates/server/src/main/resources/archetype-resources/src/main/java/__packageInPathFormat__/worldmanagement/service/WorldmanagementRestService.java

```diff
@@ -0,0 +1,41 @@
+package ${package}.worldmanagement.service;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Named;
+
+/**
+ * Simple class to test REST Service.
+ */
+@ApplicationScoped
+@Named
+@javax.inject.Named
+@Path("customermanagement/v1")
+@Consumes(MediaType.APPLICATION_JSON)
+@Produces(MediaType.APPLICATION_JSON)
```

Comment:
Again a quite heavy mix of annotations. Some are considered, some not. I would also for REST services go with spring annotations.

Reply:
We are not getting an agreement on this and should discuss this in a meeting and not here. Malte, Jan-Gerrit and others should also be on board. Ideally we ask our users. A key principle of devonfw was to focus on official standards and I am not giving up on this easy as I am fully convinced that this is the right way to go.

Reply:
As already stated above we are pointlessly blocking progress here.
Instead of my work being wasted, I would rather merge this and welcome further PRs for improvement.
However, not much further invest expected here...